samples/integration/vcpkg-all-smoke/src/main.cpp

```diff
@@ -31,9 +31,9 @@ using namespace Azure::Security::Attestation;
 
 int main()
 {
-  auto tenantId = std::getenv("AZURE_TENANT_ID");
-  auto clientId = std::getenv("AZURE_CLIENT_ID");
-  auto clientSecret = std::getenv("AZURE_CLIENT_SECRET");
+  auto tenantId = GetEnvHelper::GetEnv("AZURE_TENANT_ID");
```

Comment:
I dislike bringing in an extra concept for the samples.
I think the samples should use `getenv()` which has the standard signature, and handle nullptrs returned from it.

Samples get turned into production code, and customers should have one thing less from us to depend on, and we don't want to support an extra thing.

As customers read the sample, their attention should be drawn to SDK concepts, not on "What's that GetEnvHelper thing?"

Reply:
I am willing to revert these changes and add the appropriate #pragmas to suppress the MSVC warning. But that is going to be a LOT more code and will be just as bad w.r.t. introducing new concepts..

In my very strong opinion, headers which globally modify the behavior of the compiler should be avoided at all costs, that's what get_env.hpp does (it is the equivalent of having a using namespace declaration in a header - it poisons every consumer of that header). 

If all that get_env.hpp did was to remove the single warning on `std::getenv`, that would be one thing, but it doesn't do that - instead it disables the "you used an unsafe C runtime library API" warning globally. And that is arguably worse than introducing a new way of getting environment variables.

Reply:
Oh, and `GetEnvHelper::GetEnv` has one other important behavior that `std::getenv` does not: It ensures that environment variables that are required for the samples are defined and throws an exception if they are not.

Reply:
Ok - alternate solution. Move the _DISABLE_SECURE_CRT to the cmakefiles for each of the samples and remove the reference to get_env.hpp.